arc-versions.sh

```diff
@@ -136,7 +136,7 @@ do
     fi
 
     # Skip checkout of Linux if neither uClibc nor Glibs are being built
-    if [ "${DO_GLIBC}" = "no" ] && [ "${DO_UCLIBC}" = "no" ]
+    if [ "${tool}" = "linux" ] && [ "${DO_GLIBC}" = "no" ] && [ "${DO_UCLIBC}" = "no" ]
```

Comment:
This change is making the check even more restrict. Wasn't the behaviour before the change to `continue` with or without the _tool_ being _linux_?

Reply:
Correct, prior to the change the behavior was always to continue, which would lead to all repositories being skipped for git updates. According to the comment and in close similarity to checks a few lines above, I think the intention was to only skip updating the linux tree.

Reply:
@abrodkin Could you check, either by looking or actually running it on our build servers, that if this change is OK? If so, then we can merge it.

Reply:
> Correct, prior to the change the behavior was always to continue, which would lead to all repositories being skipped for git updates. According to the comment and in close similarity to checks a few lines above, I think the intention was to only skip updating the linux tree.

@dirker Could you update your commit message with what you just explained?

Reply:
@shahab-vahedi, done, and rebased against arc-dev.